### ISSUE TITLE
Add E-Mail templates with custom attachments to be created and sent from within SORMAS

### ISSUE THREAD

**Issue by DavidBaldsiefen** [hzi-braunschweig/SORMAS-Project#4286]
### Situation Description

A lot of users regulary send emails to cases and contacts.

### Feature Description

Implement a feature where document templates can be used to generate emails, which can then be attached with other generated content (e.g. quarantine orders).
Ideally, those emails could be directly sent from within SORMAS. 

### Additional Information

For further questions, contact @heineltp

Old issue in german: #4278

**Comment by heineltp** [comment]
Thank you @DavidBaldsiefen, 
Additional Information or Featuredescription:
The Sending of Mails out of Sormas seams to be the easiest way.
Otherweise the Sending via Outlook is a possibility.

**Comment by Marko-ilmkreis** [comment]
This might be a duplicate of #3544

**Comment by syntakker** [comment]
@Marko-ilmkreis no, this is okay. #3544 is a collection of findings, whereas this is a defined issue to implement in a sprint.

**Comment by OrgaLandkreisTuebingen** [comment]
The district of Tübingen is also looking forward to this feature!

**Comment by Lisa-GAmannheim** [comment]
The health department of Mannheim needs this feature too! This is very important and helpful to us.

**Comment by Marko-ilmkreis** [comment]
@DavidBaldsiefen @syntakker What is the current status of this feature? Is there any chance to get it implemented soon?
